spec/fixtures/vcr/works_with_where.yml

```diff
@@ -19,31 +19,49 @@ http_interactions:
       message: 
     headers:
       date:
-      - Wed, 27 Jul 2016 06:01:15 GMT
+      - Mon, 11 Dec 2017 07:47:02 GMT
       server:
       - Apache/2.4.6 (Red Hat Enterprise Linux) OpenSSL/1.0.1e-fips mod_auth_kerb/5.4
         mod_wsgi/3.4 Python/2.7.5
       expires:
-      - Wed, 27 Jul 2016 06:01:45 GMT
+      - Mon, 11 Dec 2017 07:47:32 GMT
       vary:
       - Accept,Cookie
       last-modified:
-      - Thu, 21 Jul 2016 12:24:12 GMT
+      - Tue, 26 Sep 2017 08:04:24 GMT
       allow:
       - GET, POST, PUT, PATCH, HEAD, OPTIONS
       cache-control:
       - max-age=30
       x-frame-options:
       - SAMEORIGIN
-      connection:
-      - close
       transfer-encoding:
       - chunked
       content-type:
       - application/json
     body:
       encoding: UTF-8
-      string: '{"count":0,"next":null,"previous":null,"results":[]}'
+      string: '{"count":25,"next":"https://pdc.host.dev.eng.pek2.redhat.com/rest_api/v1/releases/?active=false&page=2","previous":null,"results":[{"release_id":"rcmtools-1.5@rhel-7","short":"rcmtools","version":"1.5","name":"RCM
```

Comment:
You sure you want to use these internal identifiers?

Reply:
This is a dev env, and the same as previous cache address, so I think that's fine to use this.